integration/vault_test.go

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+//go:build requires_docker
+
+package integration
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/grafana/e2e"
+	e2edb "github.com/grafana/e2e/db"
+	hashivault "github.com/hashicorp/vault/api"
+	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/mimir/integration/e2emimir"
+)
+
+func TestVaultTokenRenewal(t *testing.T) {
+	s, err := e2e.NewScenario(networkName)
+	require.NoError(t, err)
+	defer s.Close()
+
+	// Initialize Vault
+	vault := e2e.NewHTTPService(
+		"vault",
+		"hashicorp/vault:1.13.2",
+		nil,
+		e2e.NewHTTPReadinessProbe(8200, "/v1/sys/health", 200, 200),
+		8200,
+	)
+	vault.SetEnvVars(map[string]string{"VAULT_DEV_ROOT_TOKEN_ID": "dev_token"})
+	require.NoError(t, s.StartAndWaitReady(vault))
+
+	cli, err := hashivault.NewClient(&hashivault.Config{Address: fmt.Sprintf("http://%s", vault.HTTPEndpoint())})
+	require.NoError(t, err)
+
+	cli.SetToken("dev_token")
```

Comment:
Can we extract this string into `const` to show that this value and value on line 31 are expected to be the same?